core/src/main/java/org/mapfish/print/servlet/job/PrintJob.java

```diff
@@ -172,15 +179,19 @@ public final PrintJobResult call() throws Exception {
       if (Thread.currentThread().isInterrupted()) {
         LOGGER.info(
             "Print job canceled {}\n{}", this.entry.getRequestData(), this.entry.getReferenceId());
-        this.metricRegistry.counter(getClass().getName() + ".canceled").inc();
+        this.metricRegistry
+            .counter(
+                MetricRegistry.name(
+                    getClass().getSimpleName(), "ExceptionCanceledReportGeneration"))
```

Comment:
What exception for?
```suggestion
                    getClass().getSimpleName(), "ReportGenerationCanceled"))
```

Reply:
Exception in line 178. The goal is to remove ambiguity with org.mapfish.print.servlet.MapPrinterServlet#cancelPath